src/main/java/io/conduit/SparkUtils.java

```diff
@@ -0,0 +1,34 @@
+package io.conduit;
+
+import org.apache.spark.sql.SparkSession;
+
+public final class SparkUtils {
+    private SparkUtils() {
+
+    }
+
+    public static SparkSession create(String name, DestinationConfig config) {
+        String catalogName = config.getCatalogName();
+        System.setProperty("aws.region", config.getS3Region());
```

Comment:
what if the connector is restarted, would this still be set?

Reply:
Yes, because it runs every time the connector runs.